src/VisualStudio/Core/Def/FindReferences/Contexts/WithoutReferencesFindUsagesContext.cs

```diff
@@ -113,6 +107,11 @@ protected override async ValueTask OnDefinitionFoundWorkerAsync(DefinitionItem d
                             cancellationToken).ConfigureAwait(false);
                         entries.AddIfNotNull(entry);
                     }
+
+                    foreach (var metadataLocation in definition.MetadataLocations)
```

Comment:
is it possible to have one source location, and many metadata locations?  if so, we wouldn't get into here.

Reply:
Ah, good point. I missed a condition